apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -508,6 +510,16 @@ function initializeBlocklyWrapper(blocklyInstance) {
 
     const trashcan = new CdoTrashcan(workspace);
     trashcan.init();
+
+    workspace.registerToolboxCategoryCallback(
+      'PROCEDURE',
+      procedureFlyoutCallback
+    );
+
+    // Customize function defintion blocks
+    Blockly.blockly_.Blocks[
```

Comment:
Thinking in the context of the work we were doing to not directly override core blockly fields -- is the blockly team "ok" with this sort of direct override approach with individual blocks (or at least with individual properties of existing blocks)?

Reply:
Yes, this is "ok". In a perfect world (ie. if we were building the platform from scratch), we'd use an entirely different name for our version of the block, but `procedures_defnoreturn` is used in thousands of places throughout our code base and in countless student projects.